langchain4j-mistral-ai/src/main/java/dev/langchain4j/model/mistralai/MistralAiChatModel.java

```diff
@@ -48,19 +51,55 @@
 public class MistralAiChatModel implements ChatModel {
 
     private final MistralAiClient client;
-    private final String modelName;
-    private final Double temperature;
-    private final Double topP;
-    private final Integer maxTokens;
     private final Boolean safePrompt;
     private final Integer randomSeed;
-    private final ResponseFormat responseFormat;
 
     private final Integer maxRetries;
     private final Set<Capability> supportedCapabilities;
 
+    private ChatRequestParameters defaultRequestParameters;
+
+    private MistralAiChatModel(MistralAiChatModelBuilder builder) {
```

Comment:
```suggestion
    public MistralAiChatModel(MistralAiChatModelBuilder builder) {
```

To allow extending this class